service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -107,11 +110,14 @@ public void sendNotificationForC2Upload(final C2UploadedEvent event) {
     }
 
     @EventListener
-    public void sendNotificationForOrder(final GeneratedOrderEvent event) {
+    public void sendNotificationsForGeneratedOrder(final GeneratedOrderEvent event) {
         EventData eventData = new EventData(event);
 
-        sendOrderNotificationForLocalAuthority(eventData.getCaseDetails(), eventData.getLocalAuthorityCode(),
+        sendGeneratedOrderNotificationToLocalAuthority(eventData.getCaseDetails(), eventData.getLocalAuthorityCode(),
             event.getMostRecentUploadedDocumentUrl());
+
+        sendOrderIssuedNotificationToHmctsAdmin(eventData.getCaseDetails(), eventData.getLocalAuthorityCode(),
```

Comment:
*court admin? or do we use hmcts admin in other places?

Reply:
We use HmctsAdmin everywhere